components/styles.css

```diff
@@ -155,15 +155,15 @@ p {
 
 @media screen and (min-width: 480px) {
     h1 {
-        font-size: 90pt;
+        font-size: 90px;
```

Comment:
the font sizes should usually be multiples of 8, not sure why its 90px - are these from the designs?